histolab/__init__.py

```diff
@@ -15,7 +15,3 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ------------------------------------------------------------------------
-
-# Versioning convention
-# https://www.python.org/dev/peps/pep-0440/
-__version__ = "0.5.1"
```

Comment:
Well the `__version__` alone doesn't make sense to maintain if you are not using anyware. 
But we can refactor a bit, use here the import of the metadata and in all the other places that we need to know the histolab's version we can just import `__version__`. 

I see poetry itself it's using this approach because it is using in some other places https://github.com/python-poetry/poetry/blob/master/src/poetry/__version__.py#L16

Reply:
mmm ok, that could also be a solution. Let's conclude with some refactoring